en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/01_adding_dataobjects_to_the_schema.md

```diff
@@ -299,17 +292,15 @@ By default, four loaders are provided to you to help gather specific classnames:
   - `include: [ 'src/Model/*.model.php' ]`
   - `include: [ 'somevendor/somemodule: src/Model/*.php' ]`
 
+> [!NOTE]
+> `exclude` directives will always supersede `include` directives.
```

Comment:
Moved this from below to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.